config/layout.yml

```diff
@@ -28,24 +37,42 @@ Layout/AccessModifierIndentation:
   Enabled: true
   EnforcedStyle: outdent
 
+# Our predominant style of writing multiline arrays is:
+#
+# my_array = [
+#   element_1,
+#   element_2,
+# ]
+#
+# even_in_a_method_call([
+#   element_1,
+#   element_2,
+# ])
+#
 # Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here! Suggest enabling this with
-# "EnforcedStyle: consistent".
 Layout/FirstArrayElementIndentation:
-  Description: >-
-    Checks the indentation of the first element in an array
-    literal.
-  Enabled: false
+  EnforcedStyle: consistent
 
+# Our predominant style of writing multiline hashes is:
+#
+# my_hash = {
+#   key_1: value_1,
+#   key_2: value_2,
+# }
+#
+# even_in_a_method_call({
+#   key_1: value_1,
+#   key_2: value_2,
+# })
+#
 # Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here! Suggest enabling this with
-# "EnforcedStyle: consistent".
 Layout/FirstHashElementIndentation:
-  Description: 'Checks the indentation of the first key in a hash literal.'
-  Enabled: false
+  EnforcedStyle: consistent
```

Comment:
Does this cause any problems for code like:

```
even_in_a_method_call(key_1: value_1,
                      key_2: value_2)
```

or:

```
      responses = [{ id: "100000001",
                     submission_time: "2020-04-30T12:00:00+01:00",
                     region: "Greater London",
                     question: "What is my question?",
                     question_format: "In writing, to be read out at the press conference",
                     name: "John Smith",
                     email: "john@example.com",
                     phone: "07123456789" },
                   { id: "100000002",
                     submission_time: "2020-04-30T12:00:00+01:00",
                     region: "Yorkshire and the Humber",
                     question: "What is your question?",
                     question_format: "Record a video, to be shown as part of the press conference",
                     name: "Joe Bloggs",
                     email: "joe@example.com",
                     phone: "07987654321" }]
```

Reply:
[Should do...](https://docs.rubocop.org/en/latest/cops_layout/#layoutfirsthashelementindentation)

> The `consistent` style enforces that the first key in a hash
> literal where the opening brace and the first key are on
> separate lines

I didn't see anything surprising when I trialled it.

Reply:
Great 👍